src/pages-and-resources/discussions/app-config-form/apps/lti/LtiConfigForm.jsx

```diff
@@ -1,15 +1,14 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import PropTypes from 'prop-types';
 import { FormattedMessage, injectIntl, intlShape } from '@edx/frontend-platform/i18n';
 import { Card, Form, Hyperlink } from '@edx/paragon';
 import { useFormik } from 'formik';
 import * as Yup from 'yup';
 
 import messages from './messages';
-import AppConfigFormDivider from '../shared/AppConfigFormDivider';
 
 function LtiConfigForm({
-  appConfig, app, onSubmit, intl, formRef, title,
+  appConfig, app, onSubmit, intl, formRef, title, handleConfigError,
```

Comment:
I think we probably want to name this new prop `onValidationError` - in general we use `on*` as the prefix for handlers when we pass them into components.  The corresponding handler declared in DiscussionsSettings.jsx would then be `handleValidationError`.